include/padscore/wpad.h

```diff
@@ -54,23 +54,39 @@ typedef enum WPADButton
    WPAD_BUTTON_HOME                    = 0x8000,
 } WPADButton;
 
+typedef enum WPADNunchukButton
+{
+   WPAD_NUNCHUK_STICK_EMULATION_LEFT  = 0x0001,
+   WPAD_NUNCHUK_STICK_EMULATION_RIGHT = 0x0002,
+   WPAD_NUNCHUK_STICK_EMULATION_DOWN  = 0x0004,
+   WPAD_NUNCHUK_STICK_EMULATION_UP    = 0x0008,
+} WPADNunchukButton;
```

Comment:
C is `0x4000`, Z is `0x2000`

Reply:
> 
> 
> C is `0x4000`, Z is `0x2000`

Hello @QuarkTheAwesome, you are telling me that the same value in the Wii Remote button mask is also in the Nunchuk mask with the same value?

Current `WPADButton` enum:
```c
typedef enum WPADButton
{
   WPAD_BUTTON_LEFT                    = 0x0001,
   WPAD_BUTTON_RIGHT                   = 0x0002,
   WPAD_BUTTON_DOWN                    = 0x0004,
   WPAD_BUTTON_UP                      = 0x0008,
   WPAD_BUTTON_PLUS                    = 0x0010,
   WPAD_BUTTON_2                       = 0x0100,
   WPAD_BUTTON_1                       = 0x0200,
   WPAD_BUTTON_B                       = 0x0400,
   WPAD_BUTTON_A                       = 0x0800,
   WPAD_BUTTON_MINUS                   = 0x1000,
   WPAD_BUTTON_Z                       = 0x2000,
   WPAD_BUTTON_C                       = 0x4000,
   WPAD_BUTTON_HOME                    = 0x8000,
} WPADButton;
```

So `hold` and `nunchuck.hold` both have C and Z?